### PR TITLE
add arm64 support

### PR DESCRIPTION
**Pull request by k4nzdroid**

### CODE REVIEW

File: agent/.env.example [hunk 1]
```diff
@@ -1,3 +1,10 @@
+# version of the cita-monitor
+VERSION=v0.5
```
Comment (rainchen) [inline agent/.env.example]: remove this var

File: agent/.env.example [hunk 2]
```diff
@@ -1,3 +1,10 @@
...
+
+# architecture that the cita-monitor agent run on
+# supported ARCH: amd64, arm64
+ARCH=arm64
```
Comment (rainchen) [inline agent/.env.example]: should use  x86 by default

File: agent/cita_exporter/Dockerfile.amd64 [hunk 3]
```diff
@@ -1,16 +1,20 @@
 FROM ubuntu:18.04
```
Comment (rainchen) [inline agent/cita_exporter/Dockerfile.amd64]: how about name it as "amd64.Dockerfile" ?

File: agent/cita_exporter/Dockerfile.arm64 [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+RUN apt-get update && apt-get install -y  \
+	  python3 \
+	  python3-pip \
+	  libcurl4-openssl-dev \
+    && apt-get clean \
+    && apt-get autoclean \
+    && rm -rf /var/lib/apt/lists/*
+
+ADD ./cita-cli-arm64 /bin/cita-cli
```
Comment (rainchen) [inline agent/cita_exporter/Dockerfile.arm64]: I think we can put the bin file cita-cli-arm64 into a docker image(compare to https://hub.docker.com/r/leeyr/cita-cli ), then copy it in Dockefile

File: agent/docker-compose.yml [hunk 5]
```diff
@@ -37,15 +37,13 @@ services:
       - "RABBIT_CAPABILITIES=bert,no_sort"
 
   citamon_agent_cita_exporter:
-    build: ./cita_exporter
-    image: citamon/agent-cita-exporter
+    image: citamon/agent-cita-exporter:20.2.2
```
Comment (rainchen) [inline agent/docker-compose.yml:40]: why using tag `20.2.2`? the latest release version of CITA is `v20.2.0`

Comment (rainchen) [reply agent/docker-compose.yml:40]: @0xA0000

Comment (k4nzdroid) [reply agent/docker-compose.yml:40]: It's the version number of cita-cli, and I think it should be. The tag represents which version of a tool that we use to collect metrics in containers. Like we did in rabbitmq-exporter and process-exporter and host-exporter.

Comment (rainchen) [reply agent/docker-compose.yml:40]: we need to get repo "citamon/agent-cita-exporter" source opened, put it into a sub dir

File: agent/docker-compose.yml [hunk 6]
```diff
@@ -15,7 +15,7 @@ services:
       - citamon-agent-net
 
   citamon_agent_process_exporter:
-    image: ncabatoff/process-exporter:0.4.0
+    image: citamon/agent-process-exporter:0.4.0
```
Comment (rainchen) [inline agent/docker-compose.yml:18]: we need to get repo "citamon/agent-process-exporter" source opened, put it into a sub dir

File: agent/docker-compose.yml [hunk 7]
```diff
@@ -27,7 +27,7 @@ services:
       - citamon-agent-net
 
   citamon_agent_rabbitmq_exporter:
-    image: kbudde/rabbitmq-exporter:1.0.0-RC
+    image: citamon/agent-rabbitmq-exporter:1.0.0-RC
```
Comment (rainchen) [inline agent/docker-compose.yml:30]: we need to get repo "citamon/agent-rabbitmq-exporter" source opened, put it into a sub dir

**Review by rainchen — CHANGES_REQUESTED** [review]
please update

File: agent/cita_exporter/Dockerfile [hunk 8]
```diff
@@ -1,16 +1,25 @@
-FROM ubuntu:18.04
+FROM rust:1.48 AS builder
```
Comment (rainchen) [inline agent/cita_exporter/Dockerfile:2]: should add a comment for what is this sage used for, for example: `# build cita-cli bin`

File: agent/cita_exporter/Dockerfile [hunk 9]
```diff
@@ -1,16 +1,25 @@
...
+	&& tar -xzvf musl-1.1.19.tar.gz \
+	&& cd musl-1.1.19/ \
+	&& ./configure && make && make install \
+	&& ln -sf /usr/local/musl/bin/musl-gcc /usr/local/bin/musl-gcc
+COPY ./cita-cli.git /build/cita-cli.git
+WORKDIR /build/cita-cli.git/cita-cli
+RUN rustup target add x86_64-unknown-linux-musl \
+	&& RUSTFLAGS='-A deprecated' cargo install --target x86_64-unknown-linux-musl --path .
+
+FROM ubuntu:18.04 AS final
```
Comment (rainchen) [inline agent/cita_exporter/Dockerfile:15]: should add a comment for what is this sage used for, for example: `# install requirements for cita_monitor_agent.py`

**Review by rainchen — CHANGES_REQUESTED** [review]
please update

**Review by rainchen — APPROVED** [review]

**Pull request merged**